android/expoview/src/main/java/host/exp/exponent/RNObject.java

```diff
@@ -91,6 +89,15 @@ public String version() {
     }
   }
 
+  public static Object versionedEnum(String sdkVersion, String clazz, String value) {
+    try {
+      return new RNObject("com.facebook.react.common.LifecycleState").loadVersion(sdkVersion).rnClass().getDeclaredField("RESUMED").get(null);
+    } catch (IllegalAccessException | NoSuchFieldException e) {
+      EXL.e(TAG, e);
+      return null;
```

Comment:
I'm a little bit afraid returning `null` may become a problem (like if we tried to pass `null` to a method expecting an integer). Would throwing a readable exception here make sense, or will the next exception be readable enough?

Reply:
Honestly, I would fully support just throwing RuntimeException here. This is something that should never happen and we'd love to see this as soon as possible.

However, some might argue that if it can work quite predictably in here with just printing an error, that would be enough.

As of your original question, I've notices that this is general pattern in RNObject to just print Exception and live with that. I guess in some cases returning null will just postpone error to some other place. Sometimes it will prevent crash and sometimes it will just obfuscate error message.